src/main/java/pl/targosz/homework/FileHowManyCharacters.java

```diff
@@ -0,0 +1,11 @@
+package pl.targosz.homework;
+
+import java.io.File;
+
+public class FileHowManyCharacters {
+    public static void main(String[] args) {
+        File file = new File("E:\\KURS JAVA\\Workspaceofficial\\JavaRepository\\src\\main\\java\\pl\\targosz\\homework\\resources\\data.txt");
```

Comment:
U mnie to nie dziala, bo nie mam "E:\\KURS JAVA\\WorkspaceOficcial[..]" Nie używaj ścieżki absolutnej - użyj relatywnej (względnej).

https://jaktestowac.pl/lesson/pt1-mk6-s05-05/